spyder/utils/vcs.py

```diff
@@ -118,7 +157,6 @@ def get_hg_revision(repopath):
 def get_git_revision(repopath):
     """
     Return Git revision for the repository located at repopath
-    

```

Comment:
Not sure why this was removed as it appears unrelated, but a space between the one line summary and the main docstring body is required per PEP257.